tests/scripts/thread-cert/message.py

```diff
@@ -156,6 +156,14 @@ def assertMleMessageIsType(self, command_type):
         assert(self.mle.command.type == command_type)
 
     def assertMleMessageContainsTlv(self, tlv_class_type):
+        """To confirm if Mle message contains the  TLV.
```

Comment:
nit: remove the extra space before `TLV` ?
update `TLV` to `specified TLV type`?

Reply:
fixed